configs/jsx.js

```diff
@@ -1,13 +1,8 @@
 import reactPlugin from 'eslint-plugin-react';
 
 export default [
+  reactPlugin.configs.flat.recommended,
```

Comment:
I think this is a good change. We can always override the recommended setup if we need to.